### ISSUE TITLE
Missing word - Comparison with the 2013 WCAG2ICT Note

### ISSUE THREAD

**Issue by NoahGresser** [w3c/wcag2ict#472]
The draft reads:

> - New terms from WCAG 2.1 and 2.2
>   - dragging movements, encloses, focus indicator, minimum bounding box, pointer input, process, single pointer, state, status message were added to [Glossary Items that Apply to All Technologies](https://wcag2ict.netlify.app/#glossary-items-that-apply-to-all-technologies)
>   - motion animation, perimeter, region, and user inactivity were added to [Glossary Items Used only in AAA Success Criteria](https://w3c.github.io/wcag2ict/#glossary-items-used-only-in-aaa-success-criteria)

Please insert "and" before "status message" in the first bullet to match the formatting in the second bullet.

**Comment by maryjom** [comment]
@NoahGresser Thank you for your comment, we have addressed this editorial in PR #472 which will be merged once I get a WCAG2ICT editor to approve the pull request.

**Comment by ChrisLoiselle** [comment]
> @NoahGresser Thank you for your comment, we have addressed this editorial in PR #472 which will be merged once I get a WCAG2ICT editor to approve the pull request.

@maryjom I believe you meant https://github.com/w3c/wcag2ict/pull/474?

**Comment by maryjom** [comment]
The missing word has been added and merged into the editor's draft. To read the change in-context, see the [Comparison with the 2013 WCAG2ICT Note](https://w3c.github.io/wcag2ict/#comparison-with-the-2013-wcag2ict-note) section. Closing as completed.
